lib/src/list_extensions.dart

```diff
@@ -252,6 +252,23 @@ extension ListExtensions<E> on List<E> {
     }
     return true;
   }
+
+  /// Returns an iterable whose elements are contiguous [slice]s of [this].
+  ///
+  /// Each slice is a view of this list [length] elements long, except for the
+  /// last one which may be shorter if [this] contains too few elements. Each
+  /// slice begins after the last one ends.
+  ///
+  /// As with [slice], these slices are backed by this list, which must not
+  /// change its length while the views are being used.
+  ///
+  /// For example, `[1, 2, 3, 4, 5].slices(2)` returns `[[1, 2], [3, 4], [5]]`.
```

Comment:
I'd say it returns `([1, 2], [3, 4], [5])`, the returned value is not a list, and iterables print themselves using `(...)`.